addons/resource.language.en_gb/resources/strings.po

```diff
@@ -64,7 +72,7 @@ msgstr ""
 
 #: Unknown
 msgctxt "#9"
-msgid "Kodi media center"
+msgid "Kodi Media Center"
```

Comment:
This was correct before. It should be "Kodi media center" because "media center" is not part of the name, just descriptive text. For example "Kodi flavored bread" or "Kodi franchise".